src/java/fr/paris/lutece/portal/service/message/AdminMessage.java

```diff
@@ -60,8 +60,8 @@
     private String _strTarget;
     private int _nType;
     private boolean _bCancel;
-    private Object [ ] _messageArgs;
-    private Map<String, Object> _requestParameters;
+    private transient Object [ ] _messageArgs;

```

Comment:
This does not make the class properly serializable. On reactivation, these fields will be `null` and the information will be lost.

Reply:
will be rollbacked